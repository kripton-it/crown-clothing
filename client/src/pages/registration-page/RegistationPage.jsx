import React from "react";

import SignInContainer from "../../components/sign-in/SignInContainer";
import SignUpContainer from "../../components/sign-up/SignUpContainer";

import { RegistrationPageContainer } from "./RegistrationPageStyles";

const RegistationPage = () => {
  return (
    <RegistrationPageContainer>
      <SignInContainer />
      <SignUpContainer />
    </RegistrationPageContainer>
  );
};

export default RegistationPage;
