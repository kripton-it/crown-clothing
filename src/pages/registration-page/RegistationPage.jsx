import React from "react";

import SignIn from "../../components/sign-in/SignIn";
import SignUp from "../../components/sign-up/SignUp";

import { RegistrationPageContainer } from "./RegistrationPageStyles";

const RegistationPage = () => {
  return (
    <RegistrationPageContainer>
      <SignIn />
      <SignUp />
    </RegistrationPageContainer>
  );
};

export default RegistationPage;
