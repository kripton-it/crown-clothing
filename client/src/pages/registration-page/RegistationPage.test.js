import React from "react";
import { shallow } from "enzyme";

import RegistationPage from "./RegistationPage";

it("expect to render RegistationPage page", () => {
  expect(shallow(<RegistationPage />)).toMatchSnapshot();
});
