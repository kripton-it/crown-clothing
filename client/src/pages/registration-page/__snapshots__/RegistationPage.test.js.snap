// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render RegistationPage page 1`] = `ShallowWrapper {}`;
